AdminUi/src/AdminUi/Dockerfile

```diff
@@ -1,61 +1,42 @@
+#See https://aka.ms/customizecontainer to learn how to customize your debug container and how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
 FROM mcr.microsoft.com/dotnet/aspnet:7.0 AS base
 WORKDIR /app
 EXPOSE 80
 
 FROM mcr.microsoft.com/dotnet/sdk:7.0 AS build
-
 WORKDIR /src
 COPY ["Directory.Build.props", "."]
+COPY ["Modules/Directory.Build.props", "Modules/"]

```

Comment:
Wtf. Was this actually missing? I wonder why it even built before.

Reply:
This file was re-generated using Visual Studio. I'm unsure how the old one worked, but it does sound weird that it would compile without that folder.

Reply:
It's a file (`Directory.Build.props`). But it's weird nonetheless.